app/web/features/userQueries/useLiteUsers.ts

```diff
@@ -0,0 +1,64 @@
+import { reactQueryRetries } from "appConstants";
+import { liteUserKey, liteUsersKey } from "features/queryKeys";
+import { RpcError, StatusCode } from "grpc-web";
+import { GetLiteUsersRes, LiteUser } from "proto/api_pb";
+import { useQuery } from "react-query";
+import { service } from "service";
+
+import { userStaleTime } from "./constants";
+
+// React Query typically retains the last successful data until the next successful fetch
+function useLiteUsers(ids: (number | undefined)[]) {
+  const nonFalseyIds = ids?.filter((id): id is number => !!id);
+  const query = useQuery<GetLiteUsersRes.AsObject, RpcError>({
+    queryKey: liteUsersKey(nonFalseyIds),
+    queryFn: () => {
+      const result = service.user.getLiteUsers(nonFalseyIds);
+      return result;
+    },
+    staleTime: userStaleTime,
+    enabled: nonFalseyIds.length > 0, // run only if there are valid liteUserIds
+    retry: (failureCount, error) => {
+      // don't retry if the user isn't found
+      return (
+        error.code !== StatusCode.NOT_FOUND && failureCount < reactQueryRetries
+      );
```

Comment:
I added this in as I realized it was probably a good idea, no need to call again if user not found, and defining the retryQueries allows me to reference that in the test (we seem to be doing this elsewhere).